src/makeInfinityAdder.js

```diff
@@ -5,6 +5,29 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const addr = (args) => {
+    if (args === 0) {
```

Comment:
The condition `if (args === 0)` is incorrect because `0` is a valid number that should be added to the sum. This condition should be removed or modified to correctly handle `0` as an input.